src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const priceDay = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days >= 7) {
+    return days * priceDay - bigDiscount;
+  }
+
+  if (days >= 3) {
+    return days * priceDay - smallDiscount;
+  }
+
+  return days * priceDay;
```

Comment:
You use `days * priceDay` 3 times, follow DRY - don't repeat yourself, and store it in the variable